RevenueCatUI/Templates/V2/ViewHelpers/BadgeModifier.swift

```diff
@@ -118,6 +125,7 @@ fileprivate extension View {
                 VStack(alignment: .leading) {
                     VStack {
                         ComponentsView(componentViewModels: badge.badgeViewModels, onDismiss: {})
+                            .frame(maxWidth: .infinity)
```

Comment:
Something changed somewhere and the top/bottom edge-to-edge badges were not filling the whole width, this fixes that.